src/PrestaShopBundle/Resources/config/services/adapter/product.yml

```diff
@@ -496,4 +497,4 @@ services:
     arguments:
       - '@prestashop.adapter.product.repository.product_multi_shop_repository'
       - '@prestashop.adapter.product.stock.repository.stock_available_multi_shop_repository'
-      - '@prestashop.adapter.shop.repository.shop_repository'
+      - '@prestashop.adapter.shop.repository.shop_repository'
```

Comment:
apparently this file is missing an empty line at the end

Reply:
fixed